.ci/configure_sharp.sh

```diff
@@ -193,16 +196,17 @@ verify_sharp() {
         echo "FAIL"
         exit 1
     fi
+echo "::endgroup::"
```

Comment:
Move GH specific formatters into a variable and use it here (e.g. maybe useful if we'll use another CI framework)